.eslintignore

```diff
@@ -3,4 +3,6 @@ storybook-static/**/*.js
 
 tsup.config.ts
 playwright.config.ts
-__playwright-tests__
\ No newline at end of file
+__tests__
```

Comment:
How do you feel about just calling the dir `tests`? I dunno that the underscores add anything.